_wasm_crypto/src/lib.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+use derive_more::{From, Into};
+use js_sys::Uint8Array;
+use wasm_bindgen::prelude::*;
+
+mod digest;
+
+#[wasm_bindgen]
+#[derive(Clone, Into, From)]
+pub struct DigestContext(digest::Context);
```

Comment:
In the gen code, this is missing the JSDoc, we should have rust doc blocks for publish structures like this as they will flow through to the intellisense for the library.